lib/Varien/Db/Adapter/Pdo/Mysql.php

```diff
@@ -131,7 +131,7 @@ class Varien_Db_Adapter_Pdo_Mysql extends Zend_Db_Adapter_Pdo_Mysql implements V
     /**
      * Cache frontend adapter instance
      *
-     * @var Zend_Cache_Core
+     * @var Mage_Core_Model_Cache
```

Comment:
in theory any Mage_ shouldn't be here but... I don't know, I think we can forget about this

Reply:
Yeah, I wouldn't worry about this as I'm pretty sure there's already Mage specific stuff there. Plus the while Varien lib is really only used for Mage.